uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -34,7 +34,7 @@ <h4 class="modal-title">
       </ui-select>
     </div>
     <div
-      ng-class="{'chr_disabled': !$ctrl.request.contact_id}"
+      ng-class="{'chr_disabled': (!$ctrl.request.contact_id || $ctrl.loading.postContactSelection)}"
```

Comment:
no, cannot use existing properties as this is a different user action in which ui has to be loaded/visible post contact selection